src/pycram/world_concepts/world_object.py

```diff
@@ -1135,6 +1149,11 @@ def set_joint_position(self, joint_name: str, joint_position: float) -> None:
         :param joint_name: The name of the joint
         :param joint_position: The target pose for this joint
         """
+        if (self.joints[joint_name].has_limits and
+                (not self.joints[joint_name].lower_limit <= joint_position <= self.joints[joint_name].upper_limit)):
+            joint_position = np.clip(joint_position, self.joints[joint_name].lower_limit,
```

Comment:
Please check if this works with continuous joints like the wrists of the PR2

Reply:
Done